### PR TITLE
fix: move `send all` button in mobile

### PR DESCRIPTION
**Pull request by shahin-hq**
## Summary
Closes: https://app.clickup.com/t/86dux5vtt

## Checklist

-   [ ] My changes look good in both light AND dark mode
-   [ ] The change is not hardcoded to a single network, but has multi-asset in mind
-   [ ] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Comment by ItsANameToo** [comment]
also some failing e2e @shahin-hq

**Review by samharperpittam — CHANGES_REQUESTED** [review]
Can we remove the "Available" text and see if it fits okay so the Send All text is on a single line please?
image

Design
image

File: src/domains/transaction/components/AddRecipient/AddRecipient.tsx [hunk 1]
```diff
@@ -352,16 +352,35 @@ export const AddRecipient: VFC<AddRecipientProperties> = ({
...
-									<Amount value={+remainingNetBalance} ticker={ticker} showTicker={false} />)
+							<span className="items-centers flex w-full justify-between">
+								<span>
+									<span>{t("COMMON.AMOUNT")}</span>
+									{isSenderFilled && !!remainingNetBalance && (
+										<span
+											data-testid="AddRecipient__available"
+											className="ml-1 text-theme-secondary-500 dark:text-theme-secondary-700"
+										>
+											<Amount value={+remainingNetBalance} ticker={ticker} showTicker={false} />)
```
[the remark this answers is not in the document]
Comment (shahin-hq) [reply src/domains/transaction/components/AddRecipient/AddRecipient.tsx:363]: Fixed. He has parenthesis in the example, so I kept them

[the remark this answers is not in the document]
Comment (shahin-hq) [reply src/domains/transaction/components/AddRecipient/AddRecipient.tsx]: Should be good now

**Review by samharperpittam — CHANGES_REQUESTED** [review]
Can we make the "Send all" text Dark-Navy-Blue - 600 in dark mode please.

**Review by samharperpittam — APPROVED** [review]

**Pull request merged**